float64 speed
int16 time
float64 test1
float64 test2
int16 inttest1
int16 inttest2
float64 rel_v
float64 ego_velocity
float64 lookforward_curvature
float64 target_velocity
float64 start_acceleration
float64 target_length
float64 followcar_veloicty
float64 followcar_length
